### PR TITLE
Update messages.json

### PR DESCRIPTION
**Pull request by cyanotyp**
added missing german translations

### CODE REVIEW

File: src/_locales/de/messages.json [hunk 1]
```diff
@@ -100,15 +115,81 @@
...
+    "description": "Name that matches for all the stores in the browser."
+  },
+
+  "container_default": {
+    "message": "Default",
+    "description": "Name of the default store in the browser."
+  },
+
+  "container_private": {
+    "message": "Private browsing",
```
Comment (ysard) [inline src/_locales/de/messages.json:189]: If this is justified in the German version of Firefox, could you translate these 3 sections please? 
(container_all, container_default, container_private)

File: src/_locales/de/messages.json [hunk 2]
```diff
@@ -118,7 +199,7 @@
   },
 
   "tooltip_cFpiDomain": {
-    "message": "First-Party Isolation verhindert, dass Drittanbieter Cookies verwenden, um den Benutzer auf verschiedenen Websites zu identifizieren (siehe Optionen).",
```
Comment (ysard) [inline src/_locales/de/messages.json]: Could you review the German version of this translation which has been deleted please?

**Review by ysard — CHANGES_REQUESTED** [review]
Hi, thank you very much for these translations!
I made 2 request changes, and after that i will merge this PR ;)
PS: If you want to view entries that are in English version of messages.json and that are not in the German one, on Linux:
diff src/_locales/de/messages.json src/_locales/en/messages.json | grep ": {"

File: src/_locales/de/messages.json (hunk 2 above)
Comment (cyanotyp) [reply src/_locales/de/messages.json]: reverted the accidently deleted text

**Comment by cyanotyp** [comment]
added the missing translations, for "Private Browsing" there is no common german translation, a word-wise translation does not make sense in german :)

**Comment by ysard** [comment]
Thank you very very much, I'm back and I integrate all this in the next version :)

**Pull request merged**